fuel-tx/src/checked_transaction.rs

```diff
@@ -1,856 +1 @@
-//! A checked transaction is type-wrapper for transactions which have been checked.
```

Comment:
Too bad git wasn't smart enough to detect this file was moved instead of created in fuel-vm, that way it could preserve some of the history.

Reply:
Yep, too many modifications.